lib/app.js

```diff
@@ -38,7 +38,9 @@ function getLogFile(options) {
 }
 
 function getLogStream(options) {
-  return fse.createWriteStream(getLogFile(options), { flags: 'a' });
+  const logFile = getLogFile(options);
+  fse.ensureDirSync(path.dirname(logFile));
```

Comment:
Why is this necessary?  There's an `await fse.ensureDir(config.logs.logsDir);` in the `initConfiguration` (`server.js`, line 88).

Reply:
yes, but if there is already a logger passed via config, the logs directory might not exist (yet) and then it will not be initialized.